src/components/SelectionList/Search/ReportListItem.tsx

```diff
@@ -137,6 +140,7 @@ function ReportListItem<TItem extends ListItem>({
             <View style={styles.flex1}>
                 {!isLargeScreenWidth && (
                     <ExpenseItemHeaderNarrow
+                        id={item.reportID ?? ''}
```

Comment:
I dislike the `?? ''` - if there is no id then there is no id. Can you just accept an optional inside `ExpenseItem` and simply handle this somehow?

Why I dislike this, is that if there is no reportID then setting `''` as id changes nothing, this is simply an error path in the app and should be handled accordingly

Reply:
done